reference/5.1/Microsoft.PowerShell.Core/About/about_Character_Encoding.md

```diff
@@ -0,0 +1,230 @@
+---
+title: about_Character_Encoding
+description: Describes how PowerShell uses character encoding for input and output of string data.
+ms.date: 10/21/2020
+Locale: en-US
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_character_properties?view=powershell-5.1&WT.mc_id=ps-gethelp
+schema: 2.0.0
+---
+# about_Character_Encoding
+
+## Short description
+Describes how PowerShell uses character encoding for input and output of string
+data.
+
+## Long description
+
+Unicode is a worldwide character-encoding standard. The system uses Unicode
+exclusively for character and string manipulation. For a detailed description
+of all aspects of Unicode, refer to
+[The Unicode Standard](https://www.unicode.org/standard/standard.html).
+
+Windows supports Unicode and traditional character sets. Traditional character
+sets, such as Windows code pages, use 8-bit code values or combinations of
+8-bit values to represent the characters used in a specific language or
+geographical region.
+
+PowerShell uses a Unicode character set by default. However, several cmdlets
+have an **Encoding** parameter that changes the encoding to a different
+character set. This parameter allows you to choose the specific the character
+encoding you need for interoperability with other systems and applications.
+
+The following cmdlets have the **Encoding** parameter:
+
+- Microsoft.PowerShell.Management
+  - Add-Content
+  - Get-Content
+  - Set-Content
+- Microsoft.PowerShell.Utility
+  - Export-Clixml
+  - Export-Csv
+  - Export-PSSession
+  - Format-Hex
+  - Import-Csv
+  - Out-File
+  - Select-String
+  - Send-MailMessage
+
+## The byte-order-mark
+
+The byte-order-mark (BOM) is a _Unicode signature_ in the first few bytes of a
+file or text stream that indicate which Unicode encoding is being used. For
+more information, see the
+[Byte order mark](https://en.wikipedia.org/wiki/Byte_order_mark) article in
+Wikipedia.
+
+In Windows PowerShell, any Unicode encoding, except `Utf7`, always creates a
```

Comment:
Lots of commas.  Maybe "..., Unicode encoding other than `Utf7` creates a BOM."  This also gets rid of "any", "except", and "always". ;)

Reply:
In the list below, `Utf7` is `UTF7`.  I'd suggest we be consistent (several other places and instances of this).  Not sure which one it should be,  It looks like in 5.1, it's all uppercase, but in 7.x, it's largely all lowercase except for NoBOM and BOM.  None have `Utf7` cased as first upper only.

Reply:
Fixed